mitodl/ocw-hugo-themes
# add missing facet values

### What are the relevant tickets?
closes https://github.com/mitodl/hq/issues/4025

### Description (What does it do?)
Resource type "Recitation Notes" and Topics "Russian" are not selectable because they are not valid parameters this fixes the issue.


### How can this be tested?
- set SEARCH_API_URL=https://open.mit.edu/api/v0/search/
- Run yarn start www
- Start a browser that ignores cors following the instructions here https://alfilatov.com/posts/run-chrome-without-cors/
- Go to http://localhost:3000/search/
- Verify that you can select resource type "Recitation Notes" and Topics "Russian" in both the course search and the content file search
